src/frontend/src/components/public/PIAFormTabs/review/index.tsx

```diff
@@ -344,7 +344,8 @@ const PIAReview = ({ printPreview }: IReviewProps) => {
         </>
       ) : pia?.status === PiaStatuses.EDIT_IN_PROGRESS ||
```

Comment:
This isn't about the changes you made, but I don't like how many nested ternaries we are using.

Reply:
yes I agree. many nested ternaries are indeed impacting the code's clarity and maintainability.